pages/wishlist.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Shoplify | Wishlist</title>
+    <link rel="stylesheet" href="/css/styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
+      integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="https://chalkui.netlify.app/css/styles.css" />
+  </head>
+  <body>
+    <!-- Header Navigation -->
+    <header>
+      <nav class="navbar box-shadow">
+        <div class="left-navbar">
+          <div class="logo"><a href="/index.html">Shoplify</a></div>
+        </div>
+        <div class="search-box">
+          <i class="fas fa-search"></i>
+          <input class="search" placeholder="Search" />
+        </div>
+        <ul class="right-navbar">
+          <li class="NoShow">
+            <button class="btn btn-link btn-style">
+              <a href="#">Login</a>
+            </button>
+          </li>
+          <li class="NoShow">
+            <a href="#"><i class="fa-2x far fa-heart"></i></a>
+          </li>
+          <li class="notification NoShow">
+            <a href="#"
+              ><i class="fas fa-shopping-cart fa-2x"></i
+              ><span class="badge">0</span></a
```

Comment:
The badge is lower than expected. Adjust the `top` property

Reply:
Thanks For Pointing It  Done the Changes